package.json

```diff
@@ -9,79 +9,70 @@
     "lint:nofix": "vue-cli-service lint --no-fix",
     "test:unit": "vue-cli-service test:unit --testPathPattern",
     "test:unit-silent": "vue-cli-service test:unit --silent",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand",
-    "lernaupdate": "node_modules/.bin/lernaupdate"
+    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
   },
   "dependencies": {
-    "@babel/compat-data": "^7.9.0",
-    "@mdi/font": "^5.0.45",
+    "@babel/compat-data": "^7.19.1",
+    "@mdi/font": "^5.9.55",
     "array.prototype.move": "0.0.4",
-    "axios": "^0.21.1",
-    "core-js": "^3.6.4",
+    "axios": "^0.27.2",
+    "core-js": "^3.25.2",
     "country-list": "^2.2.0",
-    "http-status-codes": "^1.3.2",
     "lodash": "4.17.21",
     "provinces": "^1.11.0",
-    "regenerator-runtime": "^0.13.3",
-    "register-service-worker": "^1.6.2",
-    "sbc-common-components": "2.4.43",
-    "uuid": "^8.3.1",
-    "v-mask": "^2.1.0",
-    "vue": "2.6.12",
+    "regenerator-runtime": "^0.13.9",
+    "sbc-common-components": "3.0.1",
+    "uuid": "^9.0.0",
+    "vue": "^2.7.10",
     "vue-affix": "^0.5.2",
-    "vue-router": "^3.1.6",
+    "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
-    "vuelidate": "^0.7.5",
-    "vuetify": "^2.2.15",
-    "vuex": "^3.1.3"
+    "vuelidate": "^0.7.7",
+    "vuetify": "^2.6.10",
+    "vuex": "^3.6.2"
   },
   "devDependencies": {
     "@babel/core": "^7.11.4",
-    "@storybook/addon-a11y": "^6.1.16",
-    "@storybook/addon-essentials": "^6.1.16",
-    "@storybook/addon-knobs": "^6.1.16",
-    "@storybook/addon-links": "^6.1.16",
-    "@storybook/addon-storysource": "^6.1.16",
-    "@storybook/preset-scss": "^1.0.3",
-    "@storybook/vue": "^6.4.22",
-    "@types/jest": "^25.1.4",
-    "@types/lodash": "^4.14.149",
+    "@mdx-js/react": "^2.1.3",
+    "@storybook/addon-essentials": "^6.5.12",
+    "@storybook/mdx2-csf": "^0.0.3",
```

Comment:
This package, and `@mdx-js/react` above, are workarounds for Storybook build errors.

Ref: https://github.com/storybookjs/storybook/issues/18094#issuecomment-1134326831